Userland/Services/WebContent/WebContentConsoleClient.cpp

```diff
@@ -49,21 +49,20 @@ void WebContentConsoleClient::handle_input(String const& js_source)
 
     auto result = script->run();
 
+    if (result.value().has_value())
+        print_html(JS::MarkupGenerator::html_from_value(*result.value()));
```

Comment:
The result.is_abrupt() case is removed from this function. 
Does it still print an error if you get one from user input?

Reply:
Yes, it still print an error in case of input:
<img width="523" alt="Screen Shot 2022-10-13 at 23 37 58" src="https://user-images.githubusercontent.com/26363017/195693488-38862e96-01a6-4aec-8c15-63ffb86c2d3b.png">